<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поставщик</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_name"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_mail"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_full_name"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="le_phone"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Почта</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Сайт</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Полное имя</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Телефон</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="le_site"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="le_note"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Заметка</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="lw_payment">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QListWidget" name="lw_shipping">
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Способы оплаты</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Способы доставки</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Принять</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Страна</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="cb_country"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_acc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_can()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_add_payment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_add_shipping()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_del_payment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>-36</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_del_shipping()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_acc()</slot>
  <slot>ui_can()</slot>
  <slot>ui_add_payment()</slot>
  <slot>ui_add_shipping()</slot>
  <slot>ui_del_payment()</slot>
  <slot>ui_del_shipping()</slot>
 </slots>
</ui>
